src/theme.c

```diff
@@ -627,6 +627,26 @@ entry(struct theme *theme, const char *key, const char *value)
 			theme->window_inactive_button_close_unpressed_image_color);
 	}
 
+	/* window drop-shadows */
+	if (match_glob(key, "window.active.shadow.radius")) {
+		theme->window_active_shadow_radius = atoi(value);
+	}
+	if (match_glob(key, "window.inactive.shadow.radius")) {
+		theme->window_inactive_shadow_radius = atoi(value);
+	}
+	if (match_glob(key, "window.active.shadow.color")) {
+		parse_hexstr(value, theme->window_active_shadow_color);
+	}
+	if (match_glob(key, "window.inactive.shadow.color")) {
+		parse_hexstr(value, theme->window_inactive_shadow_color);
+	}
+	if (match_glob(key, "window.active.shadow.inset")) {
+		theme->window_active_shadow_inset = atof(value);
+	}
+	if (match_glob(key, "window.inactive.shadow.inset")) {
+		theme->window_inactive_shadow_inset = atof(value);
+	}
```

Comment:
`atof()` can now be replaced with `set_float()` or `set_double()` via `common/parse-double.h`.